contracts/eosio.system/src/peer_keys.cpp

```diff
@@ -44,7 +44,7 @@ peer_keys::getpeerkeys_res_t peer_keys::getpeerkeys() {
    constexpr size_t max_return = 50;
 
    getpeerkeys_res_t resp;
-   resp.reserve(max_return);
+   resp.reserve(50);
```

Comment:
Why not use `max_return` so that the number 50 isn't repeated a bunch of times?

Reply:
https://github.com/eosnetworkfoundation/eos-system-contracts/pull/185/commits/d632fdedc1b846fb6f50a5685ce847e15177831b